packages/textlint/src/options.ts

```diff
@@ -80,7 +92,9 @@ export const options = optionator({
             option: "format",
             alias: "f",
             type: "String",
-            description: "Use a specific output format.",
+            description: `Use a specific output format.
+                             Available formatter          : ${concatFormatterList(getFormatterList())}
+                             Available formatter for fixer: ${concatFormatterList(getFixerFormatterList())}`,
```

Comment:
> Available formatter for fixer

Or 

> Available formatter for --fix

Reply:
Choose `--fix` as `--dry-run` option uses it. Please review updated pull request. Thank you.